exporters/otlp/include/opentelemetry/exporters/otlp/otlp_log_recordable.h

```diff
@@ -67,6 +67,16 @@ class OtlpLogRecordable final : public opentelemetry::sdk::logs::Recordable
    */
   void SetBody(const opentelemetry::common::AttributeValue &message) noexcept override;
 
+  /**
+   * @brief Set the Event Id for this log.
+   * @param id the event Id to set
+   * @param name  the event name to set
+   */
+  void SetEventId(int64_t /* id */, nostd::string_view /* name */) noexcept override
```

Comment:
Did I miss anything? I can find any specification about event id in OTLP, should we add it just like [semantic convention for event attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/semantic_conventions/events.md)?